web-animations-1/Overview.bs

```diff
@@ -833,14 +833,58 @@ follows:
     <var>animation</var>, if any.
 1.  If <var>new timeline</var> is the same object as <var>old timeline</var>,
     abort this procedure.
+1.  Let |previous play state| be |animation|'s [=play state=].
+1.  Let |previous current time| be the |animation|'s [=current time=].
+1.  Let |had finite timeline| be true if |animation| has an associated
+    <a>timeline</a> that is not [=monotonically increasing=].
 1.  Let the <a>timeline</a> of <var>animation</var> be <var>new timeline</var>.
+1.  Perform the steps corresponding to the <em>first</em> matching
+    condition from the following, if any:
+
+    <div class="switch">
+
+    :   If |animation| has an associated
+        <a>timeline</a> that is not [=monotonically increasing=],
+
+    ::  1.  [=Apply any pending playback rate=] on |animation|
+        1.  Let |seek time| be zero if [=playback=] rate &ge; 0, and
+            |animation|'s <a>associated effect end</a> otherwise.
+        1.  Update the [=start time=] and [=hold time=] based on the first
+            matching condition if any
+
+            <div class="switch">
+
+            :   If <em>either</em> of the following conditions are true:
+                *    |previous playback rate| is 'running' or,
+                *    |previous playback rate| is 'finished'
+
+            ::  Set |animation|'s [=start time=] to |seek time|.
+
+            :   If <em>both</em> of the following conditions are true:
+                *    |previous playback rate| is 'paused', and
+                *    |timeline time| is resolved
+
+            ::  1.  Set |animation|'s [=hold time=] to
+                    (|timeline time| - |seek time|) * [=playback rate=]
+                1.  Make |animation|'s [=start time=] <a>unresolved</a>
+
+            </div>
+
+    :   If |had finite timeline| and |previous current time| is resolved,
+
+    ::  Set the |animation|'s [=hold time=] to |previous current time|.
```

Comment:
start_time will be resolved here if the animation is not paused. In this case hold_time set here will be cleared in the following statement (l. 879)

Reply:
Oops, this should ahve read [=current time=] and not [=hold time=].  Making the correction.